packages/backend/src/open_payments/payment/incoming/model.ts

```diff
@@ -215,6 +216,9 @@ export class IncomingPayment
     if (this.cancelledAt) {
       data.cancelledAt = new Date(this.cancelledAt).toISOString()
     }
+    if (this.senderWalletAddress) {
+      data.senderWalletAddress = this.senderWalletAddress
+    }
```

Comment:
Maybe let's leave it out of the webhook payload for now - to not confuse integrators that this is something to expect on any incoming payents